.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.2.3
+    hooks:
+    -   id: check-yaml
+-   repo: local
+    hooks:
+      - id: fmt
+        name: fmt
+        language: system
+        files: '\.rs$'
+        exclude: '.*/templates/.*\.rs$'
```

Comment:
Good catch, thanks. Just pushed an update. Related to that section, note I set the entry to `cargo fmt`, but I know you used `rustup run nightly rustfmt` [here](https://github.com/mozilla/rust-code-analysis/blob/4a899635cc7280bdc265cc2429bd164cfe61714c/.pre-commit-config.yaml#L17). Is it important here to use nightly and/or do you think it's better to run rustfmt through rustup instead of cargo?

In regards to the unrelated to the hook: yes, I mistakenly branched after pushing other changes for an unrelated PR. I'll resolve and ping you again once that PR gets into master.

Reply:
I think fmt for stable is enough, so don't need to have it for nightly (and the same for clippy).